addons/bestja_frontend_fpbz/views/snippets.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
```

Comment:
Co to za snippet? Może przydałby się jakiś komentarz albo bardziej opisowy atrybut `name`?